packages/node_modules/@ciscospark/widget-space/src/index.js

```diff
@@ -8,6 +8,12 @@ import messages from './translations/en';
 
 export {reducers};
 
+export const toTypes = {
```

Comment:
Used the `toTypes` pattern that we use in meet widget. Probably will want to consolidate these somewhere at some point 🤣

Reply:
Lol, I was secretly hoping you would come up with a better name for it. `toTypes` just sounds silly.

Reply:
Sure!